sdk/communication/communication-job-router/src/models/options.ts

```diff
@@ -187,52 +165,52 @@ export interface GetJobOptions extends JobRouterGetJobOptionalParams {}
 /**
  * Options to cancel a job.
  */
-export interface CancelJobOptions extends JobRouterCancelJobOptionalParams {}
+export interface CancelJobOptions extends JobRouterCancelJobActionOptionalParams {}
 
 /**
  * Options to accept a job.
  */
-export interface AcceptJobOptions extends JobRouterAcceptJobOptionalParams {}
+export interface AcceptJobOptions extends JobRouterAcceptJobActionOptionalParams {}
 
 /**
  * Options to decline a job.
  */
-export interface DeclineJobOptions extends JobRouterDeclineJobOptionalParams {}
+export interface DeclineJobOptions extends JobRouterDeclineJobActionOptionalParams {}
 
 /**
  * Options to complete a job.
  */
-export interface CompleteJobOptions extends JobRouterCompleteJobOptionalParams {}
+export interface CompleteJobOptions extends JobRouterCompleteJobActionOptionalParams {}
 
 /**
  * Options to close a job.
  */
-export interface CloseJobOptions extends JobRouterCloseJobOptionalParams {}
+export interface CloseJobOptions extends JobRouterCloseJobActionOptionalParams {}
 
 /**
  * Options to get router jobs.
  */
 export interface ListJobsOptions extends JobRouterListJobsOptionalParams {}
 
 /**
- * Options to get enqueued router jobs.
+ * Options to delete a job.
  */
-export interface ListEnqueuedJobsOptions extends JobRouterListEnqueuedJobsOptionalParams {}
+export interface DeleteJobOptions extends JobRouterDeleteJobOptionalParams {}
 
 /**
  * Options to register a worker.
  */
-export interface RegisterWorkerOptions extends JobRouterRegisterWorkerOptionalParams {}
+export interface RegisterWorkerOptions extends JobRouterRegisterWorkerV2OptionalParams {}
```

Comment:
Why `V2`?

Reply:
It is automatic from ADO's model.

Reply:
I don't understand... how does that work?
(It should not have a V2 in it, we need to get rid of it)

Reply:
It is from the route name we gave in the controllers, e.g.  [HttpPost(Name = "JobRouter_CreateClassificationPolicyV2")]

If we want to get rid of V2, then we need to rename the routes in our controller. I can do that.